Package.swift

```diff
@@ -26,7 +26,7 @@ let package = Package(
         .package(url: "https://github.com/kylef/PathKit.git", exact: "1.0.1"),
         .package(url: "https://github.com/SwiftGen/StencilSwiftKit.git", exact: "2.10.1"),
         .package(url: "https://github.com/tuist/XcodeProj.git", exact: "8.3.1"),
-        .package(name: "SwiftSyntax", url: "https://github.com/apple/swift-syntax.git", branch: "0.50800.0-SNAPSHOT-2022-12-29-a"),
+        .package(name: "SwiftSyntax", url: "https://github.com/apple/swift-syntax.git", from: "508.0.0"),
```

Comment:
Thanks for doing this!

Running `swift package resolve` with this branch gives me the warning:
```warning: 'package(name:url:from:)' is deprecated: use package(url:from:) instead```
It looks like we can drop the `name: "SwiftSyntax",` from the beginning of the call there.

Reply:
Since this is just a deprecation error, I didn't feel the need to update this, but I'm happy to do so. 👍